source/views/streaming/streams/list.tsx

```diff
@@ -28,6 +29,22 @@ const groupStreams = (entries: StreamType[]) => {
 	return toPairs(grouped).map(([title, data]) => ({title, data}))
 }
 
+const getEnabledCategories = (filters: ListType[]) => {
+	return filters.flatMap((filter: ListType) =>
+		filter.spec.selected.flatMap((spec) => spec.title),
```

Comment:
Can we give a `let` binding and a type to some part of this? What type is `filter.spec`? What type is `filter.spec.selected`? No need to be super terse here, imo. If `@frogpond/filter` doesn't expose the necessary types, then we should make it do so, so I'm happy to ignore this if that is the case.

I.e., can we use

```typescript
let filterSelections: /* sometype */ = filter.spec.selected;
filterSelections.flatMap((spec) => spec.title);
```

Reply:
You don't like relying on type inference here? (Just asking for clarity.)

Reply:
I am fine with type inference, but at more than 2 or 3 levels into a given type I'm left scratching my head trying to figure out which type is being used.

Reply:
Let's come back around to this once we improve the API of the filters?